docs/containers-storage.conf.5.md

```diff
@@ -34,6 +34,11 @@ The `storage` table supports the following options:
   container storage graph dir (default: "/var/lib/containers/storage")
   Default directory to store all writable content created by container storage programs.
 
+  The rootless graphroot path supports three substations:
```

Comment:
This looks fine here, but if you do "View File" from the '...' in the review page, it's all jammed up into the prior line.